### ISSUE TITLE
couple of issues GraphViz and TF issue

### ISSUE THREAD

**Issue by javismiles** [AppliedDataSciencePartners/DeepReinforcementLearning#27]
hi friends

any ideas what to do about this warning?

WARNING:tensorflow:From /home/myuser/alpha2/DeepReinforcementLearning/loss.py:15: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.

and how to install this?
`pydot` failed to call GraphViz.Please install GraphViz (https://www.graphviz.org/) and ensure that its executables are in the $PATH

thank u :)

**Comment by acautin** [comment]
you have to install graphviz using pip but also install the executable and add the bin directory to the path. 
``pip install graphviz``
then download the installer from: ``https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi``

Hope that helps.

**Comment by liamdx** [comment]
I am also experiencing this issue, installed Graphviz and PyDot via pip, also downloaded the Graphviz installer and linked to my Path environment variable. Any suggestions would be sincerely welcomed.
